pkg/testutils/helm_test.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright The Helm Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package testutils
+
+import (
+	"context"
+	"os"
+	"os/exec"
+	"testing"
+
+	v1 "k8s.io/api/core/v1"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+func RunHelmCommand(t *testing.T, args []string) (string, error) {
+	t.Helper()
+
+	kubeConfig := os.Getenv("KUBECONFIG")
+	if kubeConfig == "" {
+		t.Fatal("KUBECONFIG environment variable is not set")
+	}
+
+	cmd := exec.Command("helm", args...)
+	cmd.Env = append(os.Environ(), "KUBECONFIG="+kubeConfig)
+
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		t.Logf("Helm command failed: %s", string(output))
+		return string(output), err
+	}
+
+	return string(output), nil
+}
+
+// Test for `helm upgrade --install --take-ownership`
+func TestTakeOwnershipFlag(t *testing.T) {

```

Comment:
I don't know this is testing helm's --take-ownership flag. This is just creating a pod with helm ownerref and then checking if the pod was created with that ownerref